.doctrine-project.json

```diff
@@ -6,8 +6,8 @@
     "codePath": "/src",
     "versions": [
         {
-            "name": "1.1",
-            "branchName": "master",
```

Comment:
what about the master branch ?

Reply:
We're not working on 1.4.0 or anything ATM, which is why I haven't requested adding it.